### DISCUSSION TITLE
Installation on Mac

### DISCUSSION THREAD

**Discussion by UCAPEM-ACADEMY** [sepinf-inc/IPED#1935] [General]
Kind regards.
Has anyone installed iPED Tools on Mac? Can you guide me through the process, since I'm trying to get it to run but it gives me multiple errors.

**Comment by lfcnassif** [comment]
In theory it is possible. I already had a student that made it work partially during a weekend class, indeed.

However this project doesn't test running the tool on MacOS neither provide support for it, because of lack of resources (MacOS machines for testing and people with available time), so we don't guarantee you will get correct results running the tool on MacOS.

**Reply by lfcnassif** [reply]
Just a final comment: the project recommends running the tool on Windows or Linux. On MacOS, you will be on your own...
